testworld/README.md

```diff
@@ -11,8 +11,8 @@ Here you can create, run and test nodes with various behaviours to observe how t
 - Bernard (`hostManager.bernard`) is a special host that serves as the libp2p bootnode for the test network.
 - `hostConfig` serves as the starting point for you to define new hosts. Please check whether an existing host can be reused for your scenario before adding new ones.
 - At the start of the each test run a test config is loaded for the required Ethereum network(eg: Rinkeby or local). The host configs are defined based on this.
-- If you want to define a custom configuration(s) for your local testing copy `configs/base.json` to `configs/local/local.json` and modify the file the way you want. Testworld will automatically pick up the file. 
-- You could also replace local config depending you requirements without having to change code by changing `otherLocalConfig` field in `configs/local/local.json`, this is useful when you want to have multiple configs locally for different centrifuge/ethereum networks such as Kovan or Rinkeby. Example:
+- If you want to define a custom configuration(s) for your local testing, copy `configs/base.json` to `configs/local/local.json` and modify the file the way you want. Testworld will automatically pick up the file. 
```

Comment:
Can you add a small section in this readme on how to add a new testcase and how to run that new test using `go test` command. I think it should basically be 3 steps which answers following questions,
- (how to define a test case?)
- (How to create/set configs? refer, as its already mentioned here also)
- (How to run with `go test`?)